docs/notebooks/2_getting_started.ipynb

```diff
@@ -38,10 +38,10 @@
     "\n",
```

Comment:
<p>*case</p><p> </p><p>but I think this sentence is more confusing than helpful..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>